README.md

```diff
@@ -1,57 +1,66 @@
-# spfx-walls
+# SPFX Walls Extension
 
 ## Summary
+ This extention lock out certain user interface components so only Tenant Admins or special roles can access them. It pulls values from the extension properties defined on SharePoint. When deployed there are already some default values provided. You can edit these from the app catalog's tenant wide section. The properties follow JSON formatting, and each property is a string that needs to start and end in double quotations. This extension is intended to be deployed tenant wide.
```

Comment:
The extension do not lock certain element. It only hide element from the user interface. If the user know the link, they can still access the settings. So I'll change the "lock out" to "hide"